src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1059,9 +1086,18 @@
   <data name="Argument_GenericArgsCount" xml:space="preserve">
     <value>The number of generic arguments provided doesn't equal the arity of the generic type definition.</value>
   </data>
+  <data name="Argument_GenericArgumentCannotBeByrefOrPointerType" xml:space="preserve">
+    <value>Generic argument can't be a byref or pointer type.</value>
+  </data>
   <data name="Argument_GenericsInvalid" xml:space="preserve">
     <value>Generic types are not valid.</value>
   </data>
+  <data name="Argument_GenericParsingError" xml:space="preserve">
+    <value>Error parsing generic params spec.</value>
+  </data>
+  <data name="Argument_OpenGenericTypeNotPermitted" xml:space="preserve">
+    <value>Open Generic Type not permitted.</value>
```

Comment:
```suggestion
    <value>Open generic type not permitted.</value>
```
missed one, don't reset just for this ..

Reply:
I just pushed anyway so I'll get this one in :)